shinhye/routes/userRouter.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+
+const userController = require("../controllers/userController");
+
+router.post("/signup", userController.signUp);
+router.post("/users/login", userController.login);
```

Comment:
RESTful API를 구현하기 위한 엔드포인트 설계가 잘못된 것 같습니다.

이미 routes/index.js 파일에서 users 리소스와 관련된 엔드포인트는 127.0.0.1:3000/users...로 설정이 되어 있습니다. 그렇기 때문에 다시 /users 리소스를 명시할 필요가 없습니다.

지금 엔드포인트는 다음과 같아 리소스의 중복이 발생하고 있는 것입니다:
127.0.0.1:3000/users/users/...

따라서 해당 router에서는 users를 지워주세요!

```suggestion
router.post("/login", userController.login);
```